packages/odyssey-react-mui/src/labs/DataComponents/useFilterConversion.ts

```diff
@@ -58,32 +58,26 @@ export const useFilterConversion = ({
   );
 
   const dataTableFilters = useMemo(() => {
-    const providedFilters = filters || columns;
-    if (!providedFilters) {
-      return [];
-    }
+    const providedFilters = filters || columns || [];
+
     return providedFilters.reduce<DataFilter[]>((accumulator, item) => {
       if (typeof item === "string") {
         const foundColumn = columns?.find(
           (column) => column.accessorKey === item,
         );
-        if (foundColumn) {
-          const filter = convertColumnToFilter(foundColumn);
-          if (filter) {
-            return accumulator.concat(filter);
-          }
-        }
-      } else if ("accessorKey" in item) {
-        // Checks if it's a column
+        const filter = foundColumn && convertColumnToFilter(foundColumn);
+        return filter ? [...accumulator, filter] : accumulator;
+      }
+
+      if ("accessorKey" in item) {
         const filter = convertColumnToFilter(item);
-        if (filter) {
-          return accumulator.concat(filter);
-        }
-      } else if ("label" in item) {
-        // Checks if it's a DataFilter
-        return accumulator.concat(item);
+        return filter ? [...accumulator, filter] : accumulator;
       }
-      // If none of the conditions match, item is ignored (not mapping to undefined)
+
+      if ("label" in item) {
+        return [...accumulator, item as DataFilter];
+      }
```

Comment:
I prefer `concat` because it's more clear what's going on:
```suggestion
      if ("label" in item) {
        return accumulator.concat(item as DataFilter);
      }
```

What I mean is it's declarative. It says "concatenate this item in a new array" rather than "spread this value into a new array with an item at the end".

Why do we have `as DataFilter` here?